scripts/release.js

```diff
@@ -0,0 +1,120 @@
+;(async () => {
```

Comment:
I think this could be top-level awaited / ESM?

Reply:
I wasn't able to, I think I'd have to declare `"type": "module",` at the `package.json` level, which would have implications all the way down.

![image](https://user-images.githubusercontent.com/14095644/143937688-c25eea8a-ae1c-436e-83ee-a97f4d60921f.png)